Tests/PointFreeTests/InviteTests.swift

```diff
@@ -362,9 +362,14 @@ class InviteTests: TestCase {
       UUID(uuidString: "deadbeef-dead-beef-dead-beefdead0001")!)
 
     await withDependencies {
+      $0 = .test
```

Comment:
for a few tests I've started being more strict by reseting everything back to test values. Currently we mock quite a few things in the base test case, but I think eventually we want to get rid of all of that and start being very explicit in tests. By doing this I was able to track down quite a few subtle things in our tests.